### PR TITLE
Docs update: unique session ID for the test.

### PR DESCRIPTION
**Pull request by jeffswartz**
You must use different OpenTok sessions for the test and for the target app.

### CODE REVIEW

**Comment by odivorra** [comment]
LGTM

File: Android-Sample/README.md [hunk 1]
```diff
@@ -14,7 +13,8 @@ the session:
 * The client is unable to publish.
 
 The sample app only subscribes to the test stream. It does not subscribe to other streams in the
-session.
+test session. Use a separate OpenTok session (and session ID) to share audio-video streams between
+clients.
```
Comment (skteh) [inline Android-Sample/README.md]: I would also include the following sentence
"Do not use the test session for your actual call. Use a separate OpenTok session (and session ID) to share audio-video streams between clients."

File: iOS-Sample/README.md [hunk 2]
```diff
@@ -14,7 +13,8 @@ the session:
 * The client is unable to publish.
 
 The sample app only subscribes to the test stream. It does not subscribe to other streams in the
-session.
+test session. Use a separate OpenTok session (and session ID) to share audio-video streams between
+clients.
```
Comment (skteh) [inline iOS-Sample/README.md]: I would also include the following sentence
"Do not use the test session for your actual call. Use a separate OpenTok session (and session ID) to share audio-video streams between clients."

**Comment by skteh** [comment]
@jeffswartz , made some comments inline

**Comment by skteh** [comment]
@jeffswartz, just pinging on this.

**Pull request merged**